diff --git a/xbmc/cores/VideoPlayer/DVDCodecs/Video/AMLCodec.cpp b/xbmc/cores/VideoPlayer/DVDCodecs/Video/AMLCodec.cpp
index 500b6e3..d0dc514 100644
--- a/xbmc/cores/VideoPlayer/DVDCodecs/Video/AMLCodec.cpp
+++ b/xbmc/cores/VideoPlayer/DVDCodecs/Video/AMLCodec.cpp
@@ -247,12 +247,6 @@ static const int64_t INT64_0 = 0x8000000000000000ULL;
 #ifndef CODEC_TAG_VC_1
 #define CODEC_TAG_VC_1  (0x312D4356)
 #endif
-#ifndef VFORMAT_VP9
-#define VFORMAT_VP9    VFORMAT_UNSUPPORT
-#endif
-#ifndef VIDEO_DEC_FORMAT_VP9
-#define VIDEO_DEC_FORMAT_VP9    VIDEO_DEC_FORMAT_MAX
-#endif
 
 #define CODEC_TAG_RV30  (0x30335652)
 #define CODEC_TAG_RV40  (0x30345652)
--
