diff --git a/xbmc/cores/VideoPlayer/DVDCodecs/Video/DVDVideoCodecAmlogic.cpp b/xbmc/cores/VideoPlayer/DVDCodecs/Video/DVDVideoCodecAmlogic.cpp
index 8f32d8d..e3218bd 100644
--- a/xbmc/cores/VideoPlayer/DVDCodecs/Video/DVDVideoCodecAmlogic.cpp
+++ b/xbmc/cores/VideoPlayer/DVDCodecs/Video/DVDVideoCodecAmlogic.cpp
@@ -33,6 +33,8 @@
 #include "settings/Settings.h"
 #include "settings/AdvancedSettings.h"
 #include "threads/Thread.h"
+#include "guilib/GUIWindowManager.h"
+#include "GUIUserMessages.h"
 
 #define __MODULE_NAME__ "DVDVideoCodecAmlogic"
 
@@ -392,6 +394,8 @@ bool CDVDVideoCodecAmlogic::AddData(const DemuxPacket &packet)
       m_videoBufferPool = std::shared_ptr<CAMLVideoBufferPool>(new CAMLVideoBufferPool());
 
       m_opened = true;
+      CGUIMessage msg(GUI_MSG_PLAYBACK_STARTED, 0, 0);
+      g_windowManager.SendThreadMessage(msg);
     }
   }
 
--
